### ISSUE TITLE
Error occured: connect ECONNREFUSED x.x.x.x:2878

### ISSUE THREAD

**Issue by Rixxter** [SebastianOsinski/HomebridgePluginSamsungAirConditioner#49]
I can´t get a Token. I´m running homebridge in Docker on Raspi 3b +

Connection configuration:
 {
  pfx: <Buffer 30 82 14 df 02 01 03 30 82 14 95 06 09 2a 86 48 86 f7 0d 01 07 01 a0 82 14 86 04 82 14 82 30 82 14 7e 30 82 0e f2 06 09 2a 86 48 86 f7 0d 01 07 06 a0 ... 5297 more bytes>,
  port: 2878,
  host: 'xxxxxxx',
  rejectUnauthorized: false,
  secureProtocol: 'TLSv1_method',
  ciphers: 'ALL:@SECLEVEL=0'
}

**Comment by SebastianOsinski** [comment]
Hi @Rixxter!

Have you tried obtaining token with `--skipCertificate` option?

**Comment by Rixxter** [comment]
yes, it also does not work. I get the same result but without the pfx Buffer
